<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuracion</class>
 <widget class="QDialog" name="Configuracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>664</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Configuracion</string>
  </property>
  <widget class="QDialogButtonBox" name="Terminar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>590</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>298</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCalendarWidget" name="CalendarioFechaI">
      <property name="selectedDate">
       <date>
        <year>2017</year>
        <month>7</month>
        <day>12</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="maximumDate">
       <date>
        <year>2020</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>298</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QCalendarWidget" name="CalendarioFechaF">
      <property name="minimumDate">
       <date>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="maximumDate">
       <date>
        <year>2020</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDateEdit" name="FechaIcaja">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2020</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="FechaFcaja">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2020</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>          Fecha de Inicio de Proyecto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>          Fecha de Fin de Proyecto</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>771</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="NombreProyecto">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;NuevoProyecto1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del proyecto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Terminar</sender>
   <signal>accepted()</signal>
   <receiver>Configuracion</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Terminar</sender>
   <signal>rejected()</signal>
   <receiver>Configuracion</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
